data/systemd/Makefile

```diff
@@ -24,7 +24,11 @@ SYSTEMD_UNITS_GENERATED := $(wildcard *.in)
 SYSTEMD_UNITS = $(sort $(SYSTEMD_UNITS_GENERATED:.in=) $(wildcard *.service) $(wildcard *.timer) $(wildcard *.socket))
 
 .PHONY: all
-all: $(SYSTEMD_UNITS)
+all: $(SYSTEMD_UNITS) check
+
+.PHONY: check
+check: snapd.manage snapd.core-fixup.sh
+	if command -v shellcheck >/dev/null; then shellcheck $?; fi
```

Comment:
Please use `$<` here as check is a phony target and the concept of being newer feels weird.

Reply:
Thank you for correcting me, I obviously meant `$^` instead.